docs/Makefile

```diff
@@ -1,6 +1,6 @@
 # Global variables
 # You can set these variables from the command line.
-POETRY        := poetry
+POETRY        := PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring poetry
```

Comment:
@lersek i'd suggest just put
```make
export PYTHON_KEYRING_BACKEND = keyring.backends.null.Keyring
```
before `POETRY := poetry` . see also https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html

more readable this way.

Reply:
@tchaikov Hi Kefu, it seems like I missed your comment here; not sure how.

I think I prefer the env var being set only for poetry (and its children). Overriding the user's keyring backend is generally unjustified; this is only a workaronud for the poetry bug, so I'd like to keep it as narrowly scoped as possible. IIUC, the `export` would apply the `PYTHON_KEYRING_BACKEND` setting to anything that make runs; that's not my intent.